sphinx_gallery/gen_gallery.py

```diff
@@ -132,6 +132,12 @@ def _complete_gallery_conf(sphinx_gallery_conf, src_dir, plot_gallery,
             backreferences_warning,
             type=DeprecationWarning)
 
+    if gallery_conf.get("thumbnail_size", False):
```

Comment:
```suggestion
    if 'thumbnail_size' not in gallery_conf:
```

Reply:
Oh wait, this actually needs to be `''thumbnail_size' not in sphinx_gallery_conf`. `gallery_conf` will always get a value above from `gallery_conf = copy.deepcopy(DEFAULT_GALLERY_CONF)`